### PR TITLE
Added my Builders Page

### PR DESCRIPTION
**Pull request by cleanerzkp**
## Description
CleanShot 2024-05-12 at 18 25 36@2x
CleanShot 2024-05-12 at 18 25 45@2x

Your ENS/address:
0x872f1fFdeEe35a6074658B4FB185E2582Db7896A

### CODE REVIEW

File: packages/nextjs/app/builders/layout.tsx
Comment (technophile-04) [inline packages/nextjs/app/builders/layout.tsx:1]: Maybe lets remove this file for now, we can nicely handle this while conquering #3  

Please feel free to add metadata to your personal page 🙌

File: packages/nextjs/app/builders/page.tsx
Comment (technophile-04) [inline packages/nextjs/app/builders/page.tsx:1]: Lets remove this file for now, We can create it in #3

File: packages/nextjs/components/Header.tsx
Comment (technophile-04) [inline packages/nextjs/components/Header.tsx:1]: Lets revert this changes, lets handle it in #3 ?

File: yarn.lock
Comment (technophile-04) [inline yarn.lock:1]: Lets revert this changes 

After removing `@types/connect` and  `@types/seedrandom` from `package.json` and running `yarn install` this changes should be automatically reverted 🙌

**Review by technophile-04 — COMMENTED** [review]
Looking great @cleanerzkp !! Really nice use of SE-2 primitives like `Address` component, daisUI etc ! 

There were some changes related to #3, I think it would be nice if we handle them in separate PR 🙌

File: packages/nextjs/app/builders/0x872f1fFdeEe35a6074658B4FB185E2582Db7896A/page.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            <span className="bg-clip-text text-black dark:text-white bg-gradient-to-r from-base-400 to-base-100 via-base-150 dark:from-secondary dark:via-secondary dark:to-secondary">
+              Cleaner
+            </span>
+            <span className="space-y-1">
+              <Address address="0x872f1fFdeEe35a6074658B4FB185E2582Db7896A" />
+            </span>
+          </h1>
+          <div className="flex gap-3">
+            <a href="https://github.com/cleanerzkp" target="_blank" rel="noopener noreferrer">
+              <img
```
Comment (technophile-04) [inline packages/nextjs/app/builders/0x872f1fFdeEe35a6074658B4FB185E2582Db7896A/page.tsx:45]: The github [actions lint fails](https://github.com/BuidlGuidl/batch5.buidlguidl.com/actions/runs/9052761727/job/24884748222?pr=16) due to use html img tag 

Lets disable the eslint for this lines by adding `{/* eslint-disable-next-line @next/next/no-img-element */}` to avoid mentioning all the remotePatterns in `next.config.js`

I think having just `avatars.githubusercontent.com` in `next.config.js` is fine since it can utilized by others too.

File: packages/nextjs/app/builders/0x872f1fFdeEe35a6074658B4FB185E2582Db7896A/page.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+const CleanerPage: NextPage = () => {
+  return (
+    <div className="w-full max-w-6xl mx-auto rounded-3xl shadow-2xl flex flex-row my-10 animate-gradient-x">
+      {/* image column */}
+      <div className="w-1/2 rounded-l-3xl bg-secondary">
+        <div className="flex justify-center items-center h-full">
+          <div className="w-80 h-80 overflow-hidden rounded-full shadow-lg">
+            <Image
+              src="/thumbnail.jpg"
```
[the remark this answers is not in the document]
Comment (cleanerzkp) [reply packages/nextjs/app/builders/0x872f1fFdeEe35a6074658B4FB185E2582Db7896A/page.tsx:22]: Yes, it's for the future, although I only have basic avatar hah

File: yarn.lock
Comment (cleanerzkp) [reply yarn.lock:1]: done!

**Comment by cleanerzkp** [comment]
hey, i made the updates after the comments could you please check @technophile-04

File: yarn.lock
Comment (technophile-04) [inline yarn.lock:1]: Forgot to remove this changes 😅 

I think running `yarn install`  should delete this changes (since we have removed extra stuff pacakge.json) 

Could please run `yarn install` and commit once again, Thanks !

File: packages/nextjs/components/Header.tsx [hunk 3]
```diff
@@ -24,6 +24,11 @@ export const menuLinks: HeaderMenuLink[] = [
     href: "/debug",
     icon: <BugAntIcon className="h-4 w-4" />,
   },
+  {
+    label: "Builders",
+    href: "/builders",
+    icon: <UsersIcon className="h-4 w-4" />,
+  },
```
Comment (technophile-04) [inline packages/nextjs/components/Header.tsx:31]: Lets remove this, until we solve #3  we can add it there

**Review by cleanerzkp — COMMENTED** [review]
ran yarn install, yarn.lock seems fine

File: packages/nextjs/app/builders/0x872f1fFdeEe35a6074658B4FB185E2582Db7896A/page.tsx [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+        <meta property="og:image" content="/thumbnail.jpg" />
+        <meta property="og:type" content="website" />
+        <meta name="twitter:card" content="summary_large_image" />
+        <meta name="twitter:title" content="Cleaner - Web3 Enthusiast and Developer" />
+        <meta
+          name="twitter:description"
+          content="Learn more about Cleaner's work in blockchain technology and his contributions to decentralized applications."
+        />
+        <meta name="twitter:image" content="/thumbnail.jpg" />
+      </Head>
```
Comment (technophile-04) [inline packages/nextjs/app/builders/0x872f1fFdeEe35a6074658B4FB185E2582Db7896A/page.tsx:38]: Lol sorry this is last thing! Instead of using Head tag which is deperecated in app router 

Screenshot 2024-05-16 at 1 10 05 PM

Your could use nice utility which Scaffold ETH provides [`getMetadata`](https://github.com/BuidlGuidl/batch5.buidlguidl.com/blob/24bbf023c27b00c1b09233aff4596a8d1a0e0eec/packages/nextjs/utils/scaffold-eth/getMetadata.ts) which you can use directly by just passing title, description and res others things are handled internally 🙌

Checkout the example here https://github.com/BuidlGuidl/batch5.buidlguidl.com/blob/24bbf023c27b00c1b09233aff4596a8d1a0e0eec/packages/nextjs/app/debug/page.tsx#L5

**Comment by cleanerzkp** [comment]
added new logo will move it to different issue if cool hah
and updated metadata! 
always learning new things, thanks

**Review by technophile-04 — APPROVED** [review]
Tysm @cleanerzkp !! Looking great, just reverted the new logo commit at https://github.com/BuidlGuidl/batch5.buidlguidl.com/pull/16/commits/f8c79432e18a1feed641ad0b17ca50e065e353d5 

We have a separate issue for that #7 , lets try to collaborate their / propose this image their and see how all batch mates feel, Lets try to handle it in separate PR 🙌

**Review by technophile-04 — APPROVED** [review]
Tysm @cleanerzkp !! Looking great, just reverted the new logo commit at https://github.com/BuidlGuidl/batch5.buidlguidl.com/pull/16/commits/f8c79432e18a1feed641ad0b17ca50e065e353d5 

We have a separate issue for that #7 , lets try to collaborate their / propose this image their and see how all batch mates feel, Lets try to handle it in separate PR 🙌
